bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,187 @@
+
+What should return is a message that the key was added successfully. An example of what this should look like is below:
+
+```
+- address: bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9
+  name: test-val
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"Ayau+8f945c1iQp9tfTVaCT5lzhD8n4MRkZNqpoL6Tpo"}'
+  type: local
+```
+
+#### Get Funds
+<!-- This needs to be updated to the correct testnet and potentially need help with the correct instructions as I cant access the faucet -->
+Request Funds from the Babylon Testnet Faucet​ [here](https://faucet.devnet.babylonlabs.io/) 
+
+##### What is BLS Voting
+
+ BLS (Boneh-Lynn-Shacham) voting is used to create compact checkpoints at the end of each epoch. An epoch is a fixed period within the blockchain, defined or set by a number of blocks, during which a validator set remains consistent. This stability allows Babylon to create and submit checkpoints only once per epoch, reducing the overhead of interacting with the Bitcoin blockchain.
+
+Once an epoch begins, the validator set is responsible for block creation and validators remain constant throughout that epoch. Each validator has BLS Keys in addition to their standard validator keys. At the end of each epoch validators cast their vote using their BLS keys and the BLS public key is registered on the Babylon network. The BLS public key is registered on the Babylon network. Valid BLS signatures are then collated into a single signature which is then used for a babylon checkpoint. A checkpoint is not only made up of aggregated BLS signatures from the validator set but also a hash of the last block in the epoch and other epoch related data. These checkpoints are submitted to the BTC blockchain. providing an immutable record of Babylon's state and protecting against long-range attacks.
+
+The primary difference between BLS voting and regular voting is the signature aggregation. This allows Babylon to create very compact proofs of validator participation, which can be efficiently stored and verified on Bitcoin. This mechanism is crucial for Babylon's security model, enabling it to leverage Bitcoin's security while maintaining its own Proof-of-Stake system.
+
+#### Create BLS Key
+
+Validators are expected to submit a BLS signature at the end of each epoch. To do that, a validator needs to have a BLS key pair to sign information with.
+
+Using your address from the keyring generation run:
+`babylond create-bls-key <address>`
+
+Your address should be the one that was generated in the keyring generation step and looks something like this: `bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9`
+
+This command will create a BLS key and add it to the `~/.babylond/config/priv_validator_key.json` that was generated when `init` was run.
+
+### Run Node
+
+We are now ready to run the node.
+
+`./build/babylond start --chain-id=bbn-test-5 --home=./nodeDir --log_level=info --minimum-gas-prices=0.005ubbn`
```

Comment:
Do we need to add all of these CLI flags? Also, why are we working with the `build` directory? We can instruct users to do `install` and work with just `babylond` for the rest of the document. We can have a brief note about using `build` and specifying the `/build` directory, and let more advanced users figure out where the binary is themselves for the rest of the doc.

Reply:
Regarding the flags:
Log level can be left out.
As for gas prices, we can put this config directly into the corresponding configuration file - otherwise, the user has to specify it every time.
In the existing docs and in the infra, I see that we also use `--x-crisis-skip-assert-invariants`. Does it make sense to include it here?

Regarding build vs install:
I'd recommend install so that the user can change directories w/o worrying about the relative babylond binary path.